extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisUtils.scala

```diff
@@ -81,17 +84,19 @@ object KinesisUtils {
    *                                 the tip of the stream (InitialPositionInStream.LATEST).
    * @param storageLevel Storage level to use for storing the received objects
    *
-   * @return JavaReceiverInputDStream[Array[Byte]]
+   * @return JavaReceiverInputDStream[ Array[Byte] ]
    */
   @Experimental
   def createStream(
-      jssc: JavaStreamingContext, 
-      streamName: String, 
-      endpointUrl: String, 
+      jssc: JavaStreamingContext,
```

Comment:
Isn't this changing the API?
I believe you should make a JIRA along with this PR. It's not trivial. 
https://cwiki.apache.org/confluence/display/SPARK/Contributing+to+Spark